imports/api/users/Reserves.js

```diff
@@ -0,0 +1,23 @@
+import { SimpleSchema } from 'meteor/aldeed:simple-schema';
+
+const Schema = {};
+Schema.Reserves = new SimpleSchema({
+  token: {
+    type: String,
+    defaultValue: ['WEI'],
```

Comment:
- This is where we could set `defaultValue` to a function that returns an array [as indicated by the docs](https://github.com/aldeed/simple-schema-js#allowedvalues)
- The returned array could be populated with the `code` for desired elements of `token.js` lib. 
- Flagging here that we might have to update SimpleSchema in the future to [latest version](https://github.com/aldeed/meteor-simple-schema/blob/master/CHANGELOG.md#200):
>SimpleSchema is now an NPM package. The version numbering started over, but you can consider v0.x.x of the NPM package to be v2 of the Meteor package. The Meteor package will get critical fixes only. You should attempt to migrate to using the NPM package as soon as you can.